instances/technique/galvanicSkinResponse.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/galvanicSkinResponse",
```

Comment:
this is not a technique. The galvanic skin response which seems to be an outdated term for electrodermal activity (EDA) is a property that can be measured. EXAMPLE definition: EDA is an indirect measure of neurally mediated effects on sweat glands’ permeability, observed as changes in the resistance of the skin to a small electrical current or as differences in the electrical potential between different parts of the skin. ([source](https://www.analog.com/en/technical-articles/design-development-and-evaluation-of-a-system-to-obtain-electrodermal-activity.html))

For EBRAINS: please add this for now as term suggestion with the suggested terminology "BiologicalPhenomenon" (does not exist yet but is in prep). As for the technique: leave it out for now, but I will investigate if "epidermal electrophysiology technique" (or something like this) would be the correct superclass for any electrical measurement from the skin. @tgbugs & @UlrikeS91 TBD

Reply:
galvanicSkinResponse.jsonld deleted from techniques. [Term suggestion](https://editor.kg.ebrains.eu/instances/eef5c3b4-03d9-4645-9d9a-0164f84ed40e) added.